common/util.go

```diff
@@ -803,3 +804,54 @@ func AreKafkaPhantomStores(storeUUIDs []string) bool {
 
 	return len(storeUUIDs) == 1 && storeUUIDs[0] == KafkaPhantomExtentStorehost
 }
+
+// AreDestinationZoneConfigsEqual determines whether two zone configs have the same content
+func AreDestinationZoneConfigsEqual(left []*shared.DestinationZoneConfig, right []*shared.DestinationZoneConfig) bool {
+	if len(left) != len(right) {
+		return false
+	}
+	for _, l := range left {
+		zone := l.GetZone()
+		foundMatch := false
+		for _, r := range right {
+			if r.GetZone() == zone {
```

Comment:
btw, another way to do this would obviously be to do run through one of the slices and put it into a map; and then run through the other while looking up against the map. but i guess the zone-config list would be really small to make a difference between O(n^2) vs O(n)?